### ISSUE TITLE
Error on account without contacts

### ISSUE THREAD

**Issue by yabb85** [movim/movim#870]
If you try to use movim with an account without contact you have the following error in log:
```
[2019-09-29 20:07:37] movim.ERROR: TypeError: Argument 1 passed to explodeJid() must be of the type string, null given, called in /usr/share/movim/App/Contact.php on line 174 in /usr/share/movim/App/helpers/StringHelper.php (line 181)  [] []
[2019-09-29 20:07:38] movim.ERROR: TypeError: Argument 1 passed to explodeJid() must be of the type string, null given, called in /usr/share/movim/App/Contact.php on line 174 in /usr/share/movim/App/helpers/StringHelper.php (line 181)  [] []
[2019-09-29 20:07:38] movim.ERROR: Error: Call to a member function conferences() on null in /usr/share/movim/App/widgets/Chat/Chat.php (line 824)  [] []
```

Installed on Debian buster with official package.

**Comment by edhelas** [comment]
Could you please try with the master branch ? The Debian package is quite old

**Comment by edhelas** [comment]
Closing the ticket, was normally fixed for a while in trunk and released with 0.16
